Remotion/Web/Core/Res/Themes/NovaViso/HTML/FormGrid.css

```diff
@@ -113,26 +92,24 @@ td.formGridTopDataRow
 {
   visibility: hidden;
   transition-property: visibility;
-  transition-duration: 0;
-  transition-delay: 0;
+  transition-duration: 0s;
+  transition-delay: 0s;
   transition-timing-function: linear;
-
   content: attr(aria-label);
   white-space: pre;
   display: inline-block;
   position: absolute;
-  left: 0;
-  top: 1.2em;
-  z-index: 1000;
-  padding: 0.25em;
-  border: 1px solid Black;
-  background-color: Lightyellow;
-  box-sizing: border-box;
+  /* popup spawns at the bottom right corner of the icon's hover highlighting */
+  top: calc(var(--default-icon-size) + var(--remotion-themed-spacing-half));
+  z-index: var(--z-index-max);
+  padding: var(--remotion-themed-spacing-half);
+  border: var(--remotion-themed-alert-border);
+  background-color: var(--color-alert-background);
```

Comment:
we should make those "tooltip" instead of "alert". And I just noticed that firefox and chrome on WIndows 10 use the same tooltip style. please emulate these colors (font, border, background) for tooltips.